packages/annotated/keypoint/src/Keypoints.jsx

```diff
@@ -0,0 +1,308 @@
+import React from "react";
```

Comment:
Would a generic version that I derived from this code be useful in this package? 

https://github.com/fairinternal/sketch_rig/blob/main/ui/www/src/components/PoseEditor.tsx

Reply:
yes.. this looks great! the only thing is that I don't have typescript support set up yet in `@annotated/dev-scripts`. I'd like to add that in, but it will most likely be a follow up PR. I'll update to your version of the PoseEditor in a follow-up PR along with that (this also makes for a great test case).